docs/source/models/gbd2017/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -94,5 +98,10 @@ Folic Acid Fortification
 Desired Model Outputs
 ---------------------
 
+Stratification
+++++++++++++++
+
```

Comment:
Can we add stratification by location? * Locations: **Nigeria, India, Ethiopia**
Does the Gates Foundation want any subpopulations included? 

Can we include in the outline the list of observers needed, to be able to output the desired model outputs? Or is that in a different document?

Reply:
Hmm, I hadn't thought of it that way -- we **do** want it stratified by location as well, but usually I've seen one output.hdf file per location, and then the different column names are where the further stratification by age/sex/year happen. But perhaps it would be better to explicitly say that we're stratifying by location as well?

The list of observers should eventually go here in the concept model document -- I just haven't gotten to that part yet.

Reply:
Oh, and right now we don't have any requests for specific subpopulations, so I'm keeping it as simple as possible unless and until we hear otherwise.